core/src/main/java/uk/gov/justice/services/core/annotation/Component.java

```diff
@@ -26,6 +26,7 @@
     COMMAND_HANDLER("command", "handler", Queue.class),
     EVENT_LISTENER("event", "listener", Topic.class),
     EVENT_PROCESSOR("event", "processor", Topic.class),
+    EVENT_API("event", "api", Queue.class),
```

Comment:
I think it should be null - in this implementation the enum defines what it's listening to, and the event api will not be listening to JMS